bookserver/routers/books.py

```diff
@@ -57,20 +57,38 @@
 # :index:`todo`: **Routes for draft (instructor-only) books.**
 @router.get("/published/{course:str}/_static/{filepath:path}")
 async def get_static(course: str, filepath: str):
+    course_row = await fetch_course(course)
     filepath = safe_join(
-        settings.book_path, course, "build", course, "_static", filepath
+        settings.book_path,
+        course_row.base_course,
+        "build",
+        course_row.base_course,
+        "_static",
+        filepath,
     )
     rslogger.debug(f"GETTING: {filepath}")
-    return FileResponse(filepath)
+    if os.path.exists(filepath):
+        return FileResponse(filepath)
+    else:
+        raise HTTPException(404)
 
 
 @router.get("/published/{course:str}/_images/{filepath:path}")
```

Comment:
This seems like duplicate code -- can you chain the two decorators on the same function instead?